lib/alchemy/test_support/having_picture_thumbnails_examples.rb

```diff
@@ -385,6 +385,26 @@
           size: "160x120"
         )
       end
+
+      context "with settings indicating free height" do
+        let(:settings) do
+          {
+            crop: true,
+            size: "800x"
+          }
+        end
+
+        it "returns default thumbnail options" do
```

Comment:
Wouldn't we expect that the thumbnail is resized by the form factor of the original image instead? 
That way it would reflect what will be the output in the frontend, if we leave one dimension `nil`, no?

Reply:
The problem is that when there's no image, `image_file_height` is `nil` or `0` (not sure). It's really only for that case (or for the case one uploads an image with height `0` - but the only person doing that is probably the proverbial software tester who asks for -1 beer in a bar :)